### PR TITLE
docs(select): improve docs for the `Option` type, clarifying that `value` must be a `string`

### LINKED ISSUE #919: limel-select with multiple selection broken on phone
## Current behavior

it opens the native selector/ dropdown where you can't select multiple options and returns undefined when selecting one.
reported on the forum: https://forum.lime-crm.com/t/error-in-web-component-on-mobile/2814

## Environment

- **lime-elements** version: 26.27.4
- Framework used: stencil

### PR DESCRIPTION
**Pull request by adrianschmidt**
fix #919

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Comment by Befkadu1** [comment]
What do you think if we use this code `value={String(option.value)}` in `select.template.tsx`? If the user uses numeric values on some iOS devices, it might avoid breaks.

File: src/components/select/option.types.ts [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+
+    /**
+     * The unique value of the option. Should always be the same for any given
+     * option, regardless of localization. The type `T` defaults to `string`,
+     * but can be set to any type that extends `string` (using `Option<type>`),
+     * for example an enum of specific strings.
+     *
+     * Note the value of this property *must* be a string!
+     * If the value you wish to use is, for example, numeric, convert it to a
+     * string before sending it to limel-select. Using numeric values does work
```
Comment (Befkadu1) [inline src/components/select/option.types.ts:18]: Using numeric values work

**Comment by adrianschmidt** [comment]
> What do you think if we use this code `value={String(option.value)}` in `select.template.tsx`? If the user uses numeric values on some iOS devices, it might avoid breaks.

Hmm… yeah, since we don't actually seem to be using any data from the change-event, that might well work. I was thinking we'd have problems with comparing values and stuff, but it looks like we never actually use the value that's been set. I'll try it 👍

File: src/components/select/option.types.ts (hunk 1 above)
Comment (adrianschmidt) [reply src/components/select/option.types.ts:18]: I disagree. I'm fairly certain that "using numeric values does work in many desktop browsers" is correct. The singular "does" is referring to the verb "using" (which I assume is uncountable) not the plural noun "values".

**Comment by adrianschmidt** [comment]
> > What do you think if we use this code `value={String(option.value)}` in `select.template.tsx`? If the user uses numeric values on some iOS devices, it might avoid breaks.
> 
> Hmm… yeah, since we don't actually seem to be using any data from the change-event, that might well work. I was thinking we'd have problems with comparing values and stuff, but it looks like we never actually use the value that's been set. I'll try it 👍

No, unfortunately that doesn't seem to help 😞

**Review by Befkadu1 — APPROVED** [review]

**Pull request merged**